lighthouse-core/gather/driver.js

```diff
@@ -418,17 +418,86 @@ class Driver {
     });
   }
 
+  /**
+   * Deprecated: renamed to `evaluate`.
+   * @param {string} expression
+   * @param {{useIsolation?: boolean}=} options
+   * @return {Promise<*>}
+   */
+  async evaluateAsync(expression, options = {}) {
+    return this.evaluate(expression, options);
```

Comment:
Could drop this. if we like this, I'd follow up with a PR that changes all existing usages to this.